game/scripts/npc/items/custom/item_trumps_fists.txt

```diff
@@ -72,22 +72,22 @@
       "03"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "bonus_health"                                    "900 1125"
+        "bonus_health"                                    "1000 1250"
       }
       "04"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "bonus_mana"                                      "1000 1250"
+        "bonus_mana"                                      "750 1000"
       }
       "05"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "heal_prevent_percent"                            "-50 -75"
+        "heal_prevent_percent"                            "-75"
       }
       "06"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "heal_prevent_duration"                           "3"
+        "heal_prevent_duration"                           "1 2"
```

Comment:
Leave the heal prevention duration at 3 seconds.

Reply:
I might make it 2 3 (since status resistance)

either way the code is bugged